packages/victory-box-plot/src/helper-methods.js

```diff
@@ -368,8 +368,20 @@ const getBaseProps = (props, fallbackProps) => {
     }
   };
   const boxScale = scale.y;
+  const [minDomainY, maxDomainY] = domain.y;
   return data.reduce((acc, datum, index) => {
     const eventKey = !isNil(datum.eventKey) ? datum.eventKey : index;
+
+    // if all y points are out of bound of the domain, filter out this datum
+    const { _min, _max } = datum;
+    const underMin = (val) => val < minDomainY;
```

Comment:
you could add the existence check to these underMin / overMax methods to make the following if statements a bit more readable